Sources/Tracking/Repository/CleanupRepository.swift

```diff
@@ -7,13 +7,13 @@ internal protocol CleanupRepository: AutoMockable {
 
 // sourcery: InjectRegister = "CleanupRepository"
 internal class CioCleanupRepository: CleanupRepository {
-    private let queue: Queue
+    private let queueStorage: QueueStorage
 
-    init(queue: Queue) {
-        self.queue = queue
+    init(queueStorage: QueueStorage) {
+        self.queueStorage = queueStorage
     }
 
     func cleanup() {
-        queue.deleteExpiredTasks()
+        _ = queueStorage.deleteExpired()
```

Comment:
Can we please add a small comment here to explain why we had to use `queueStorage` directly instead of `queue`? Will be helpful for me to understand how it fixes bad memory crash and anyone looking at the code and making changes in future.

Reply:
Good suggestion, Rehan. For your reference, this comment in [opsbug ticket](https://github.com/customerio/opsbugs/issues/3618#issuecomment-1640910138) might help you understand how queueStorage fixes the issue.